README.md

```diff
@@ -1,10 +1,11 @@
 <!-- Do not translate this -->
 <details>
 <summary>
-<strong> Read this guide in other languages </strong>
+<strong> இந்த Read.md மித மொழிகளில் உள்ளன  </strong>
```

Comment:
change மற்ற i think that correct
இந்த வழிகாட்டியை மற்ற மொழிகளில் படிக்கவும்

Reply:
```suggestion
<strong>இந்த வழிகாட்டியை மற்ற மொழிகளில் படிக்கவும்</strong>
```